setup.py

```diff
@@ -1,5 +1,5 @@
 import os
-from setuptools import setup
+from setuptools import setup, find_packages
```

Comment:
Shouldn't isort be requiring these imports to be alphabetical?

```suggestion
from setuptools import find_packages, setup
```

Reply:
Ah, maybe this warning is the issue:

```
lint run-test: commands[1] | isort --check-only --diff cfgov_setup
WARNING: Unable to parse file cfgov_setup due to [Errno 21] Is a directory: '/Users/cranfills/Projects/cfgov-django-setup/cfgov_setup'
```